tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Duplicate issue - buglist shows on focus, hides on blur": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("description")
+          .click()
+          .end()
+          .sleep(500)
+          .findById("bugList")
+          .getVisibleText()
+          .then(function(text) {
+            assert.equal(text, "");
+          });
+      },
+
+      "Duplicate issue - 3 issues with url found": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("bugList")
+          .findAllByCssSelector("li a")
+          .getVisibleText()
+          .then(function(texts) {
+            assert.isAtLeast(texts.length, 3, "found 3 issues");
```

Comment:
Do we ever expect more than 3 ever?

Reply:
Sorry, I'm my mind I replied. 

We have more than 3 issues that come back in this case... As issues will be added, the number will vary, but will be more than 3.

Reply:
OK, thanks. We probably want to limit the upper bounds of what we display (and when we do, add a test for that), so we're not displaying 20+ dupe issues on a mobile device, etc.

Reply:
I've added this to #1990